kubernetes/feature/monitoring/grafana/dashboards/10826_rev1.json

```diff
@@ -0,0 +1,1091 @@
+{
```

Comment:
When testing, some dashboard had a message regarding some failure regarding some legacy thing.

Should not be blocking, anyway.